### PR TITLE
Various small improvements

### PR DESCRIPTION
**Pull request by achoum**
Small changes:

- Customize font size
- Check if timestamps are sorted when importing tfexamples
- Add FeatureSchema and IndexSchema to the API
- Allow "from_tensorflow_record" to consume a list of paths.
- Propagate works if both arguments don't have the same index values.
- Plot works with evsets having different index values.
- Speed-up of plotting with evsets with a large number of index values.

### CODE REVIEW

**Review by ianspektor — APPROVED** [review]

File: temporian/implementation/numpy/data/event_set.py [hunk 1]
```diff
@@ -340,6 +340,11 @@ def num_events(self) -> int:
             count += len(data.timestamps)
         return count
 
+    def num_indexes(self) -> int:
+        """Total number of indexes."""
```
[the remark this answers is not in the document]
Comment (achoum) [reply temporian/implementation/numpy/data/event_set.py:344]: Good point.
*Update documentation.

File: temporian/implementation/numpy/data/plotter.py [hunk 2]
```diff
@@ -267,6 +283,7 @@ def plot(
         merge: If true, plots all features in the same plots. If false, plots
             features in separate plots. merge=True on event-sets [e1, e2] is
             equivalent to plotting (e1, e2).
+        font_scale: Scalling factor for all the fonts.
```
[the remark this answers is not in the document]
Comment (achoum) [reply temporian/implementation/numpy/data/plotter.py:286]: *Done

File: temporian/implementation/numpy/data/plotter.py [hunk 3]
```diff
@@ -379,6 +391,20 @@ def plot_with_plotter(
             title = None
 
             for group_item in group.items:
+                if index not in group_item.evset.data:
+                    if group_item.feature_idx == -1:
+                        tag = "sampling"
```
[the remark this answers is not in the document]
Comment (achoum) [reply temporian/implementation/numpy/data/plotter.py:396]: Good point.
*Changed here and below

File: temporian/implementation/numpy/data/test/plotter_test.py [hunk 4]
```diff
@@ -187,6 +195,42 @@ def test_build_groups(self):
...
+        try:
+            tmp_handle = tempfile.TemporaryDirectory()
+            tmp_path = os.path.join(tmp_handle.name, "fig.png")
+
+            if interactive:
+                bokeh_export_png(plot, filename=tmp_path)
+            else:
+                plot.savefig(tmp_path)
+        except RuntimeError:
+            # Fails if chromedriver is not available.
```
[the remark this answers is not in the document]
Comment (achoum) [reply temporian/implementation/numpy/data/test/plotter_test.py:231]: It is not available in some of our internal testing setups.
I am not sure about GitHub action.

**Comment by achoum** [comment]
Thanks

**Pull request merged**